test_dbapi/test_dbapi.py

```diff
@@ -94,41 +94,36 @@ def _test_errors(self, connection: dbapi.Connection):
         with pytest.raises(dbapi.InterfaceError):
             dbapi.YdbDBApi().connect("localhost:2136", database="/local666")
 
-        cur = connection.cursor()
+        with connection.cursor() as cur:
```

Comment:
Hello, can you copy the test for the cursor with context manager instead of change current test: for test both variants: standard without context manager and new variant

Reply:
we have both variants in tests for cursor